packages/manager/modules/carrier-sip/src/cdr/cdr.html

```diff
@@ -0,0 +1,43 @@
+<section>
+    <tuc-toast-message></tuc-toast-message>
+    <h3 data-translate="carrier_sip_cdr_title"></h3>
```

Comment:
Is there a `h2` above ?

Reply:
Please refer to my previous comment: https://github.com/ovh-ux/manager/pull/1382#discussion_r330424111